Libplanet/Net/Swarm.cs

```diff
@@ -460,18 +460,23 @@ public string TraceTable()
                 BlockChain.Tip.Hash
             );
 
-            var peersWithHeight = (await DialToExistingPeers(dialTimeout, cancellationToken))
-                .Where(pp => pp.Item2.TipIndex > (initialTip?.Index ?? -1))
-                .Select(pp => (pp.Item1, pp.Item2.TipIndex))
+            var peeers = await DialToExistingPeers(dialTimeout, cancellationToken);
```

Comment:
1. How about specifying the type of `peeers`.
2. I guess `peeers` is typo.

Reply:
This was included by accident. :(   I'll fix it!